operations/helm/charts/grafana-agent/CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ internal API changes are not present.
 Unreleased
 ----------
 
+### Bugfixes
+
+- Fix a bug preventing the `.Values.configReloader.image.digest` Helm value to be correctly retrieved. (@claudioscalzo)
+- Fix a bug preventing digests to be used as labels because of their length. (@claudioscalzo)
```

Comment:
Is there a documented limit for the length of this label that we can point to? Why choose 7 characters here?

Reply:
Yes, added the documented limit to the changelog entry ✅ I chose a 7-character string because it's the standard for short SHAs